packages/shared/src/hooks/squads/useSourceModerationList.ts

```diff
@@ -98,14 +103,34 @@ export const useSourceModerationList = ({
     user,
     squad.id,
   );
-  const squadQueryKey = generateQueryKey(RequestKey.Squad, user, squad.handle);
```

Comment:
Handling squad query doesn't seem to be necessary as the feed is refetched on navigate.